From: Chip Coldwell <coldwell@redhat.com>
Date: Mon, 17 Dec 2007 07:44:36 -0500
Subject: [scsi] mpt fusion: pci ids for version 3.04.05+
Message-id: alpine.LFD.0.9999.0712170742280.16626@localhost.localdomain
O-Subject: [RHEL-5.2 PATCH] 1/3 bz253122 Update mpt fusion to version 3.04.05+
Bugzilla: 253122

Adds two vendor IDs to include/linux/pci_ids.h

 include/linux/pci_ids.h |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/include/linux/pci_ids.h b/include/linux/pci_ids.h
index 90e0a0d..7b554f2 100644
--- a/include/linux/pci_ids.h
+++ b/include/linux/pci_ids.h
@@ -1459,6 +1459,7 @@
 #define PCI_DEVICE_ID_TOSHIBA_TC86C001_MISC	0x0108
 #define PCI_DEVICE_ID_TOSHIBA_SPIDER_NET 0x01b3
 
+#define PCI_VENDOR_ID_ATTO		0x117c
 #define PCI_VENDOR_ID_RICOH		0x1180
 #define PCI_DEVICE_ID_RICOH_RL5C465	0x0465
 #define PCI_DEVICE_ID_RICOH_RL5C466	0x0466
@@ -2025,6 +2026,7 @@
 #define PCI_DEVICE_ID_FARSITE_TE1C      0x1612
 
 #define PCI_VENDOR_ID_ARIMA		0x161f
+#define PCI_VENDOR_ID_BROCADE		0x1657
 
 #define PCI_VENDOR_ID_SIBYTE		0x166d
 #define PCI_DEVICE_ID_BCM1250_HT	0x0002
